docs/source/_includes/create-genesis-block.inc

```diff
@@ -72,7 +72,7 @@ genesis block also includes the keys for the other nodes in the initial network.
         --key /etc/sawtooth/keys/validator.priv \
         -o config-consensus.batch \
         sawtooth.consensus.algorithm.name=pbft \
-        sawtooth.consensus.algorithm.version=VERSION \
+        sawtooth.consensus.algorithm.version=1.0 \
```

Comment:
Please remove the sentence that follows this code block:

"Replace ``VERSION`` with the PBFT version number."